bokeh/palettes.py

```diff
@@ -450,10 +511,10 @@ def Greys9(self):
         return ["#000000", "#252525", "#525252", "#737373", "#969696", "#bdbdbd", "#d9d9d9", "#f0f0f0", "#ffffff"]
     @property
     def Greys10(self):
-        return ['#000000', '#1c1c1c', '#383838', '#555555', '#717171', '#8d8d8d', '#aaaaaa', '#c6c6c6', '#e2e2e2', '#ffffff']
+        return ["#000000", "#1c1c1c", "#383838", "#555555", "#717171", "#8d8d8d", "#aaaaaa", "#c6c6c6", "#e2e2e2", "#ffffff"]
```

Comment:
Several of the lists used single quotes while others used double quotes.  It was partially for consistency, and partially trying to make sure there were no extra quotes somewhere leading to the extra quote above the BrBg plot on the [docs page](http://bokeh.pydata.org/en/latest/docs/reference/palettes.html) (shown below).  

As there is no functional difference, I am fine leaving it the way it was.  Is there a preference one way or the other in the style guide used for Bokeh?

<img width="295" alt="screen shot 2016-10-16 at 10 05 26 pm" src="https://cloud.githubusercontent.com/assets/9484229/19423275/b4624d24-93ec-11e6-8718-e072397046c5.png">